index.html

```diff
@@ -112,6 +112,10 @@ <h2 class="title">Form</h2>
         <label for="error_field">.input.is-error</label>
         <input type="text" id="error_field" class="input is-error" placeholder="awesome.css">
       </div>
+      <div class="field">
+        <label for="textarea_field">Textarea</label>
+        <textarea id="textarea_field" class="textarea">
```

Comment:
Close tag(`</textarea>`) is required 👍 
```suggestion
        <textarea id="textarea_field" class="textarea"></textarea>
```

Reply:
You're right, that was my bad.